### PR TITLE
Migrate JSONField to Django builtin JSONField

### PR DESCRIPTION
**Pull request by ericholscher**
This runs a migration that converts from text to jsonb.

In my local testing it converted the field properly:

```
ethicaladserver=# select targeting_parameters from adserver_flight;
    targeting_parameters
----------------------------
 {                         +
     "niche_targeting": 0.9+
 }
(1 row)

ethicaladserver=# select targeting_parameters from adserver_flight;
   targeting_parameters
--------------------------
 {"niche_targeting": 0.9}
(1 row)

```

### CODE REVIEW

File: requirements/production.txt [hunk 1]
```diff
@@ -31,7 +27,7 @@ billiard==4.2.0
     # via celery
 bleach==6.1.0
     # via -r base.in
-celery[redis]==5.4.0
+celery==5.4.0
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply requirements/production.txt:30]: Going back to pip-compile.

File: requirements/base.in [hunk 2]
```diff
@@ -44,7 +44,6 @@ IP2Proxy
 # Countries helper used in ad targeting
 django-countries
 
-jsonfield
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply requirements/base.in:47]: Hrm, yea. Weird edge case.

[the remark this answers is not in the document]
Comment (ericholscher) [reply requirements/base.in]: I guess we'll keep it installed until we squash these migrations?

**Review by davidfischer — APPROVED** [review]
Not sure what the deal is with simple history, but sans that this looks good.

**Comment by davidfischer** [comment]
The test failures seem to be a result of https://github.com/jazzband/django-simple-history/issues/1181 which is still open. We may have to either [ignore historical changes on jsonfields](https://django-simple-history.readthedocs.io/en/stable/historical_model.html#excluded-fields), stop adding a change reason (remove calls to `update_change_reason`), or wait on updating our JSONFields.

**Comment by ericholscher** [comment]
FWIW, I never look at `update_change_reason`, though that's mostly because simple history doesn't do diffs, so the history is mostly useless to understand what changed.

**Comment by davidfischer** [comment]
I created a PR https://github.com/readthedocs/ethical-ad-server/pull/924 to remove `update_change_reason` so we can move forward with this.

**Comment by ericholscher** [comment]
Cool. Let's plan to get it merged and deployed next week. 👍

**Pull request merged**